src/bindings/python/flux/util.py

```diff
@@ -609,6 +609,43 @@ def filter_empty(self, items):
         #  Return new format string created from pruned format_list
         return "".join(self._fmt_tuple(*x) for x in format_list)
 
+    def print_items(self, items, no_header=False, pre=None, post=None):
+        """
+        Handle printing a list of items with the current format.
+
+        First pre-process format using ``items`` to remove any empty
+        fields, if requested. (The pre-processing step could be extended
+        in the future.)
+
+        Then, generate a header unless supperess_header is True.
```

Comment:
typo supperess_header -> suppress_header

Reply:
oh wait, changed variable while working on this, is it "no_header" now?

Reply:
Thanks, I rebased and fixed this issue (squashing the resuilt)